userprofile/urls.py

```diff
@@ -12,6 +12,5 @@
         name="privacy_notice"),
     url(r'^accounts/$', views.ProfileView.as_view(),
         name="profile"),
-    url(r'^i18n/setlang', views.LanguageView.set_language),
-    url(r'^i18n/', include('django.conf.urls.i18n')),
+    url(r'^setlang', views.LanguageView.set_language, name="set_language"),
```

Comment:
These could be named `set_user_language` and `set_enroll_language`, so it would be always clear on which action is supposed to happen. In the future, `set_language` will be a bit ambiguous.